### PR TITLE
Revamped listing show

### PR DESCRIPTION
**Pull request by jacksonvaldez**
Improved UI for listing show page. Fixed tests

### CODE REVIEW

**Review by croixk — APPROVED** [review]
Looks good!

**Review by Eldridge-Turambi — APPROVED** [review]
Good work Jacksoun

File: app/views/listings/show.html.erb [hunk 1]
```diff
@@ -1,17 +1,37 @@
...
-<p>Miles: <%= @listing.odometer %></p>
-<p>Color: <%= @listing.color %></p>
-<p>Interior Color: <%= @listing.interior %></p>
-<p>Price: <%= @listing.selling_price %></p>
+  tr:nth-child(even) {
+    background-color: #dddddd;
+  }
+</style>
+
+<div class="container px-4 px-lg-5 my-5">
```
Comment (tjroeder) [inline app/views/listings/show.html.erb:19]: The bootstrap layout looks a lot better.

**Review by tjroeder — APPROVED** [review]
Bootstrappin'

**Pull request merged**